Cosminis/Models/Companion.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+
+namespace Models;
+
+    public enum MoodCompanion                            
+    {
+        Happy, Sad, Angry, Tired, Anxious, Excited, Chill
+    }
+
+    public class CompanionMoodWithEnum
+    { 
+        public int CreatureId { get; set; }
+        public int UserFk { get; set; }
+        public int SpeciesFk { get; set; }
+        public string? Nickname { get; set; }
+        public MoodCompanion Mood { get; set; }
+        public int? Hunger { get; set; }
+
+        public CompanionMoodWithEnum()
+        {
+            
+        }
+       
+        public string MoodToString(MoodCompanion Mood)                                   
+        {
+            Dictionary<MoodCompanion, string> dictMood = new Dictionary<MoodCompanion, string>()
+            {
+                {MoodCompanion.Happy, "Happy"},
+                {MoodCompanion.Sad, "Sad"},
+                {MoodCompanion.Angry, "Angry"},
+                {MoodCompanion.Tired, "Tired"},
+                {MoodCompanion.Anxious, "Anxious"},
+                {MoodCompanion.Excited, "Excited"},
+                {MoodCompanion.Chill, "Chill"}
+
+            };
+
+            if(dictMood.ContainsKey(Mood))
+            {
+                return dictMood[Mood];
+            }
+            return "NotFound";
```

Comment:
Can you make this lint throw an exception?

Reply:
^